components/ipfs/browser/BUILD.gn

```diff
@@ -6,31 +6,38 @@ source_set("browser") {
     "brave_ipfs_client_updater.h",
     "features.cc",
     "features.h",
+    "ipfs_interstitial_controller_client.cc",
+    "ipfs_interstitial_controller_client.h",
     "ipfs_json_parser.cc",
     "ipfs_json_parser.h",
     "ipfs_navigation_throttle.cc",
     "ipfs_navigation_throttle.h",
+    "ipfs_not_connected_page.cc",
+    "ipfs_not_connected_page.h",
     "ipfs_service.cc",
     "ipfs_service.h",
     "ipfs_service_observer.h",
-    "translate_ipfs_uri.cc",
-    "translate_ipfs_uri.h",
```

Comment:
@bbondy has a different PR in progress for moving all files into browser folder, I'll leave it as is here, I move this one to common in this PR because I'm using a function in translate_ipfs_uri.cc in ipfs_utils which is currently under common folder.

Reply:
yeah, np, just a side note. We basically can just keep everything in `components/ipfs`